stack-lts-19.6.yaml

```diff
@@ -0,0 +1,8 @@
+resolver: lts-19.6
+packages:
+- '.'
+extra-deps: []
+flags: {}
+extra-package-dbs: []
+extra-include-dirs:
+    - /Library/Developer/CommandLineTools/SDKs/MacOSX12.3.sdk/usr/include/ffi/
```

Comment:
This should not be part of the repository.

What problem are you having that this solves? Perhaps there is a better solution. We never needed this before.

Reply:
It complains about the missing `ffi` includes.

Reply:
I'm sorry, but I don't think we can have this in the configuration file. It points to an arbitrary place that makes sense on your machine, but is not part of the repository, nor is it relevant for linux and windows platforms.

Can you point to what exactly the error is? We can discuss this in a meeting if it is more convenient.

Reply:
I see your point. On the other hand, we need a solution for the mac targets. You can just ignore them for now. I'll dive into this problem later.

Reply:
The client builds fine on an intel mac.

So it is not a general mac issue, it is a specific issue either to M1 macs, or to your setup.

I am not sure what you mean by "just ignore them for now". This PR cannot be merged with this line in it. Do you mean you will remove it?

Reply:
I mean I'll investigate it more and then we'll discuss it. Till then we'll not merge any of the PRs having this issue.

Reply:
removed